packages/mask/src/plugins/Avatar/Application/PersonaPage.tsx

```diff
@@ -43,21 +43,22 @@ export function PersonaPage({ onNext, onChange }: PersonaPageProps) {
     const network = socialIdentity?.identifier?.network.replace('.com', '')
     const userId = socialIdentity?.identifier?.userId
 
-    const myPersonas = usePersonasFromDB()
-    const persona = useSubscription(context.currentPersona)
-    const currentPersona = myPersonas.find(
-        (x: PersonaInformation) => x.identifier.rawPublicKey.toLowerCase() === persona?.rawPublicKey.toLowerCase(),
-    )
-
     const { value: bindingPersonas = EMPTY_LIST } = usePersonasFromNextID(
-        persona?.publicKeyAsHex,
+        socialIdentity?.publicKey,
         NextIDPlatform.NextID,
     )
     const bindingProofs = useMemo(
         () => bindingPersonas.map((x) => x.proofs.filter((y) => y.is_valid && y.platform === network)).flat(),
         [bindingPersonas],
     )
 
+    const myPersonas = usePersonasFromDB()
```

Comment:
CC @UncleBill

Reply:
@zhouhanseng You should revert changes in this file.

Reply:
got.